composer.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "goalgorilla/shariff",
+    "description": "Shariff enables website users to share their favorite content without compromising their privacy.",
```

```suggestion
    "description": "Fork of npm-asset/shariff for the Open Social Drupal distribution. Not maintained for use outside of Open Social",
```

Since we want to make this public on packagist.org so it's available for all distribution users. Lets make the purpose a bit clearer so that we don't confuse people who may be looking for the library for other purposes.